README.md

```diff
@@ -85,6 +85,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
 | [Glookast](https://www.glookast.com/) | Solving the media workflow puzzle. | `Matosinhos` |
+| [mediagaps](http://www.mediagaps.com/) | Bridging your workflow gaps. | `Porto` |
```

Comment:
Can you please review the name case?  
According to the page footer it should be `MediaGaps`.

Reply:
Yes indeed!